packages/ui/src/components/va-modal/VaModal.vue

```diff
@@ -213,8 +214,12 @@ export default defineComponent({
     watch(valueComputed, (value: boolean) => {
       if (value) {
         window.addEventListener('keyup', listenKeyUp)
+        if (props.blur) {
+          document.body.classList.add('overlay--blured')
```

Comment:
I would gently suggest moving this feature into separate block. So that features are not mixed up.

I.e.

```js
if (props.blur) {
  if (value) {
    document.body.classList.add('overlay--blured')
  } else {
    document.body.classList.remove('overlay--blured')
  }
}
```